Mathlib/Tactic/Simps/Basic.lean

```diff
@@ -269,31 +269,36 @@ syntax simpsRule := simpsRule.prefix <|> simpsRule.rename <|> simpsRule.erase <|
 syntax simpsProj := ppSpace ident (" (" simpsRule,+ ")")?
 
 /--
-This command specifies custom names and custom projections for the simp attribute `simpsAttr`.
-* You can specify custom names by writing e.g.
-  `initialize_simps_projections Equiv (toFun → apply, invFun → symm_apply)`.
-* See Note [custom simps projection] and the examples below for information how to declare custom
-  projections.
+This command allows customisation of the lemmas generated by `simps`.
+
+By default, tagging a definition of an element `myObj` of a structure `MyStruct` with `@[simps]`
+generates one `@[simp]` lemma `myObj_myProj` for each projection `myProj` of `MyStruct`. There are a
+few exceptions to this general rule:
 * For algebraic structures, we will automatically use the notation (like `Mul`)
   for the projections if such an instance is available.
 * By default, the projections to parent structures are not default projections,
   but all the data-carrying fields are (including those in parent structures).
+
+This default behavior is customisable as such:
 * You can disable a projection by default by running
   `initialize_simps_projections Equiv (-invFun)`
   This will ensure that no simp lemmas are generated for this projection,
-  unless this projection is explicitly specified by the user.
+  unless this projection is explicitly specified by the user (as in
+  `@[simps invFun] def myEquiv : Equiv _ _ := _`).
 * Conversely, you can enable a projection by default by running
-  `initialize_simps_projections Equiv (+toEquiv)`.
+  `initialize_simps_projections MulEquiv (+toEquiv)`.
```

Comment:
Hmm, can we somehow make the examples more consistent throughout? Switching between MulEquiv and Equiv and myEquiv is somewhat confusing, and somewhat suggests that changing `Equiv` is relevant for setting up `simps` on `MulEquiv`. Perhaps we could use `MulEquiv` as the running example throughout?

Reply:
What about now?